### PR TITLE
fix: broken links

### PR DESCRIPTION
**Pull request by katiegoines**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/auth/concepts/tokens-and-credentials/index.mdx [hunk 1]
```diff
@@ -258,7 +258,6 @@ Token revocation is enabled automatically in Amplify Auth. To revoke tokens you
 
 ## Next steps
 
-- [Learn how to customize the ID token](/[platform]/build-a-backend/auth/examples/customize-id-token/)
-- [Learn how to customize the access token](/[platform]/build-a-backend/auth/examples/customize-access-token/)
```
Comment (jacoblogan) [inline src/pages/[platform]/build-a-backend/auth/concepts/tokens-and-credentials/index.mdx:262]: Is there no new link for customizing the access token?

Comment (katiegoines) [reply src/pages/[platform]/build-a-backend/auth/concepts/tokens-and-credentials/index.mdx:262]: @aspittel advised to remove this link

**Review by jacoblogan — DISMISSED** [review]

File: src/pages/[platform]/deploy-and-host/fullstack-branching/cross-account-deployments/index.mdx [hunk 2]
```diff
@@ -43,13 +43,28 @@ Please refer to this Amazon CodeCatalyst [guide](https://docs.aws.amazon.com/cod
 
 ### Step 2: Deploy a fullstack Amplify Gen 2 app
 
-- Use our [**Next.js starter template**](/[platform]/start/quickstart/nextjs-pages-router/#1-create-the-repository) to create a repository in your GitHub account.
+<InlineFilter filters={["flutter", "nextjs"]}>
```
Comment (timngyn) [inline src/pages/[platform]/deploy-and-host/fullstack-branching/cross-account-deployments/index.mdx:46]: Should this filter only have `nextjs`?

```suggestion
<InlineFilter filters={["nextjs"]}>
```

File: src/pages/[platform]/deploy-and-host/fullstack-branching/cross-account-deployments/index.mdx [hunk 3]
```diff
@@ -43,13 +43,28 @@ Please refer to this Amazon CodeCatalyst [guide](https://docs.aws.amazon.com/cod
...
+ - Use our [**Next.js starter template**](/[platform]/start/quickstart/nextjs-pages-router/#1-create-the-repository) to create a repository in your GitHub account.
 - Sign in to the [AWS Management Console](https://console.aws.amazon.com/).
 - Navigate to the Amplify console and select [**Create new app**](https://us-west-2.console.aws.amazon.com/amplify/create).
 - Select the **next-pages-template** repository, then select **Next**.
 - Review the details on the **Create Git Repository** page, then select **Save and deploy**.
 - Done! You have successfully deployed a fullstack Gen 2 app. You can review the status of the app deployment in the Amplify console.
 
+ </InlineFilter>
+
+<InlineFilter filters={["angular", "android", "javascript", "react", "react-native", "swift", "vue"]}>
```
Comment (timngyn) [inline src/pages/[platform]/deploy-and-host/fullstack-branching/cross-account-deployments/index.mdx:57]: If the above inline filter wasn't supposed to have `flutter`, then we'll need to add it here too I think

```suggestion
<InlineFilter filters={["angular", "android", "javascript", "react", "react-native", "swift", "vue", "flutter"]}>
```

**Review by timngyn — APPROVED** [review]

**Review by jacoblogan — APPROVED** [review]

**Pull request merged**